api/v1alpha1/helper.go

```diff
@@ -31,7 +31,7 @@ func GetServiceAccountName(etcdObjMeta metav1.ObjectMeta) string {
 
 // GetConfigMapName returns the name of the configmap for the Etcd.
 func GetConfigMapName(etcdObjMeta metav1.ObjectMeta) string {
-	return fmt.Sprintf("etcd-bootstrap-%s", string(etcdObjMeta.UID[:6]))
+	return fmt.Sprintf("%s-config-%s", etcdObjMeta.Name, etcdObjMeta.UID[:6])
```

Comment:
As stated in another comment lets remove the suffix of UID. One can look at the owner reference and also check the name of the configmap which also has etcd-name in it.